app/backend/common/helper.py

```diff
@@ -417,8 +417,8 @@ def execute_cloud_task(params, agent_type, ec2_access_key, ec2_secret_key,
     This method instantiates celery tasks in the cloud.
     Returns return value from celery async call and the task ID
     '''
-    if 'bucket_name' not in params:
-        params['bucketname'] = ''
+    #if 'bucket_name' not in params:
+    #    params['bucketname'] = ''
```

Comment:
Why?  I have gotten crashes when we don't pass bucket name.  We "should", but don't always.